### PR TITLE
Unstable MultiHeadAttention

### PR DESCRIPTION
**Pull request by abourramouss**

### CODE REVIEW

**Comment by abourramouss** [comment]
issue #301

File: source/test-suite/test-chunk.lisp [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+        (let ((python-chunks (with-torch (x)
+                               (torch_chunk_test x chunks dim))))
+          (let ((python-chunk1 (->caten (nth 0 python-chunks)))
+                (python-chunk2 (->caten (nth 1 python-chunks)))
+                (python-chunk3 (->caten (nth 2 python-chunks))))
+            ;; Compare each chunk
+            (assert-equal
+             (:rtol 1e-5 :atol 1e-5)
+             python-chunk1
+             (proceed lisp-chunk1))
```
Comment (hikettei) [inline source/test-suite/test-chunk.lisp:29]: need !contiguous

File: source/test-suite/test-chunk.lisp [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+            (assert-equal
+             (:rtol 1e-5 :atol 1e-5)
+             python-chunk2
+             (proceed lisp-chunk2))
```
Comment (hikettei) [inline source/test-suite/test-chunk.lisp:33]: need !contiguous

**Comment by abourramouss** [comment]
some fixes needed before tests pass:

- [ ] #305 
- [ ] #304

**Pull request closed without merging**